docs/examples/dialog_box.md

```diff
@@ -68,5 +68,34 @@ def test_num_input():
     )
     response = hardpy.run_dialog_box(dbx)
     assert response == 123.0
+
+
+@pytest.mark.case_name("Radiobutton")
```

Comment:
The style of the case name must match the style above.

Reply:
Ready